### ISSUE TITLE
No STDOUT in IRuby

### ISSUE THREAD

**Issue by kylekyle** [SciRuby/iruby#78]
In IRuby `$stdout` works, but `STDOUT` doesn't:

```ruby 
STDOUT.puts 'heyo'  # nothing
$stdout.puts 'heyo' # => heyo
```

This breaks a lot of gems, including [hexdump](https://github.com/postmodern/hexdump).

**Comment by kylekyle** [comment]
Looks like `STDERR` is also broken.

**Comment by minad** [comment]
STDOUT and STDERR should not be overwritten in my opinion since they might be used by the kernel

**Comment by kylekyle** [comment]
Won't fix per #79.
